plugins/openwhisk/src/main/java/com/navercorp/pinpoint/plugin/openwhisk/interceptor/TransactionIdStartedInterceptor.java

```diff
@@ -0,0 +1,122 @@
+import com.navercorp.pinpoint.plugin.openwhisk.descriptor.DefaultMethodDescriptor;
+import scala.runtime.AbstractFunction0;
+import whisk.common.LogMarkerToken;
+
+/**
+ * @author Seonghyun Oh
+ */
+public class TransactionIdStartedInterceptor implements AroundInterceptor {
+
+    private final PLogger logger = PLoggerFactory.getLogger(TransactionIdStartedInterceptor.class);
+
+    private final TraceContext traceContext;
+    private final MethodDescriptor descriptor;
+
+    protected final boolean isDebug = logger.isDebugEnabled();
+    private final boolean isLoggingMessage;
+
+    public TransactionIdStartedInterceptor(TraceContext traceContext, MethodDescriptor descriptor) {
+        this.traceContext = traceContext;
+        this.descriptor = descriptor;
+
+        final OpenwhiskConfig config = new OpenwhiskConfig(traceContext.getProfilerConfig());
+        this.isLoggingMessage = config.isLoggingMessage();
+    }
+
+    /**
+     *
+     * @param target
+     * @param args TransactionMetadata, Identify, LogMarkerToken
+     */
+    @Override
+    public void before(Object target, Object[] args) {
+
+        AsyncContext asyncContext = AsyncContextAccessorUtils.getAsyncContext(args[0]);
+        if (asyncContext == null) {
+            logger.debug("Not found asynchronous invocation metadata {}", (LogMarkerToken)args[2]);
+            return;
+        }
+
+        Trace trace = asyncContext.continueAsyncTraceObject();
+        if (trace == null) {
+            logger.debug("trace object null");
+        }
+
+        try {
+            trace.traceBlockBegin();
+        } catch (Throwable th) {
+            if (logger.isWarnEnabled()) {
+                logger.warn("BEFORE. Caused:{}", th.getMessage(), th);
+            }
+        }
+    }
+
+
+    @Override
+    public void after(Object target, Object[] args, Object result, Throwable throwable) {
+        if (isDebug) {
+            logger.afterInterceptor(target, args, result, throwable);
+        }
+
+        AsyncContext asyncContext = AsyncContextAccessorUtils.getAsyncContext(args[0]);
+        if (asyncContext == null) {
+            logger.debug("Not found asynchronous invocation metadata");
+            return;
+        }
+        if (isDebug) {
+            logger.debug("Asynchronous invocation. asyncContext={}", asyncContext);
+        }
+
+        final Trace trace = asyncContext.currentAsyncTraceObject();
+        if (trace == null) {
+            return;
+        }
+        if (isDebug) {
+            logger.debug("Asynchronous invocation. asyncTraceId={}, trace={}", asyncContext, trace);
+        }
+        traceContext.removeTraceObject();
+
+        try {
+            final SpanEventRecorder recorder = trace.currentSpanEventRecorder();
+
+            String token = args[2].toString();
+            String message = ((AbstractFunction0) args[3]).apply().toString();
+
+            recorder.recordApi(new DefaultMethodDescriptor(token));
```

Comment:
What about method show like action_state().
I suggest that it looks good on the user side because it looks similar to other plugins.
ex)
```
            LogMarkerToken logMarkerToken = (LogMarkerToken) args[2];

            String action = logMarkerToken.action();
            String state = logMarkerToken.state()
			
```